lib/KBox/src/KMessageNMEAVisitor.cpp

```diff
@@ -25,6 +25,12 @@
 #include "KMessageNMEAVisitor.h"
 #include "util/NMEASentenceBuilder.h"
 #include "util/nmea2000.h"
+//RIGM added
+#include "tasks/NMEA2000Task.h"
+#include "KBoxDebug.h"
+
+#define IMU_IN_NMEA2K true  //variable to determine whether IMU data sent out over N2K
+unsigned int _imuSequence = 1;
```

Comment:
This is a bit of a hack and we could hide it better by making this variable a static variable in the function where it is used. We do not need anywhere else.